src/nvim/msgpack_rpc/remote_ui.c

```diff
@@ -66,8 +66,9 @@ static Object remote_ui_attach(uint64_t channel_id, uint64_t request_id,
       || args.items[2].type != kObjectTypeBoolean
       || args.items[0].data.integer <= 0 || args.items[1].data.integer <= 0) {
     api_set_error(error, Validation,
-                  _("Arguments must be a pair of positive integers "
-                    "representing the remote screen width/height"));
+                  _("Arguments must be:\n- remote screen width (integer > 0)\n"
+                    "- remote screen height (integer > 0)\n"
+                    "- whether to enable RGB capabilities (boolean)\n"));
```

Comment:
Are you sure this final newline is required? I can't find any other calls to `api_set_error` where a newline is appended to the this argument.